SofaKernel/modules/SofaDeformable/src/SofaDeformable/SpringForceField.inl

```diff
@@ -177,6 +146,27 @@ SReal SpringForceField<DataTypes>::getPotentialEnergy(const core::MechanicalPara
     return ener;
 }
 
+template<class DataTypes>
+void SpringForceField<DataTypes>::addSpringForce(Real& ener, VecDeriv& f1, const VecCoord& p1, const VecDeriv& v1, VecDeriv& f2, const VecCoord& p2, const VecDeriv& v2, sofa::Index /*i*/, const Spring& spring)
+{
+    sofa::Index a = spring.m1;
+    sofa::Index b = spring.m2;
+    typename DataTypes::CPos u = DataTypes::getCPos(p2[b])-DataTypes::getCPos(p1[a]);
+    Real d = u.norm();
+    if( spring.enabled && d<1.0e-4 ) // null length => no force
```

Comment:
I know it was in the code before, but we could replace this magic number ? 😗

Reply:
What's the good practice in this case? Is there a global constant value somewhere?

Reply:
Should I use
1. https://github.com/fredroy/sofa/blob/master/SofaKernel/modules/Sofa.Type/src/sofa/type/Vec.h#L34
2. https://en.cppreference.com/w/cpp/types/numeric_limits/epsilon
3. Another
?

Reply:
@fredroy any thoughts?